Homework/HW01.hs

```diff
@@ -0,0 +1,60 @@
+module Homework.HW01 where
+
+-- Homework Url https://www.cis.upenn.edu/~cis194/spring15/hw/01-intro.pdf
+
+-- Exercise 1 -----------------------------------------
+
+-- Get the last digit from a number
+lastDigit :: Integer -> Integer
+lastDigit num = num `mod` 10
+
+-- Drop the last digit from a number
+dropLastDigit :: Integer -> Integer
+dropLastDigit num = num `div` 10 
+
+-- Exercise 2 -----------------------------------------
+
+toRevDigits :: Integer -> [Integer]
+toRevDigits num 
+    | num > 0           = lastDigit num : (toRevDigits . dropLastDigit) num
+    | otherwise         = []
+
+-- Exercise 3 -----------------------------------------
+
+-- Double every second number in a list starting on the left.
+doubleEveryOther :: [Integer] -> [Integer]
+doubleEveryOther [] = []
+doubleEveryOther (x:y:xs) = x: 2*y : doubleEveryOther xs
+doubleEveryOther (x:_) = [x]
```

Comment:
The empty and  single-element case can be unified as they do the same thing.
```suggestion
doubleEveryOther (x:y:xs) = x: 2*y : doubleEveryOther xs
doubleEveryOther xs = xs
```

Reply:
got it, 
right this `x:_` was unnecessary here